src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
The `grid-template-columns` property is set to only one column of 200px, which doesn't seem to allow for a flexible layout. Consider using `auto-fit` or `auto-fill` combined with `minmax` for a more responsive grid that adapts to the available width while maintaining the column size.